dcos_test_utils/test_gce.py

```diff
@@ -0,0 +1,78 @@
+import dcos_test_utils.gce as gce
+import dcos_launch
+import yaml
+import os
+import pytest
+
+
+@pytest.fixture(scope='module')
+def gce_wrapper():
+    credentials_path = os.environ.get('GOOGLE_APPLICATION_CREDENTIALS', os.path.expanduser('~') + '/.gce/credentials.json')
+    return gce.GceWrapper(credentials_path)
+
+
+@pytest.fixture(scope='module')
+def config(gce_wrapper):
+    cwd = os.path.dirname(os.path.realpath(__file__))
+    with open(cwd + '/../dcos_launch/sample_configs/gce-onprem.yaml') as cfile:
+        config_content = cfile.read()
+    conf = yaml.load(config_content)
+    template_path = conf.get('template_path', cwd + '/../dcos_launch/templates/gce-template.yaml')
+    with open(template_path, 'r') as tfile:
+        template_content = tfile.read()
+    template_content = template_content.replace('${network}', conf['network'])
+    template_content = template_content.replace('${sourceImage}', conf['sourceImage'])
+    template_content = template_content.replace('${machineType}', conf['machineType'])
+    template_content = template_content.replace('${zone}', conf['zone'])
+    template_content = template_content.replace('${project_id}', gce_wrapper.project_id)
+    conf['template_content'] = template_content
+    return conf
+
+
+@pytest.fixture(scope='module')
+def deployment(gce_wrapper, config):
+    return gce.Deployment(gce_wrapper, config['deployment_name'], config['zone'])
+
+
+@pytest.fixture(scope='module')
+def instances(deployment):
+    return deployment.get_instances()
+
+
+def test_gce_init(gce_wrapper):
+    assert gce_wrapper
+
+
+def test_config(config):
+    assert config
+
+
+def test_deployment(deployment):
+    assert deployment
+
+
+def test_deploy(gce_wrapper, config):
```

Comment:
As we talked about, we don't want these tests to be apart of the unit testing experience because then people would be launching actual money-costing clusters each time they check to see if their code has proper syntax. So, lets move this whole file up a layer were it is no longer residing in the testpath as specific in tox and give it a name that doesnt start with test such as `dev_test_gce.py` to make it clear that this should not be run without an expectation of external consequences